config/locales/pt-BR/models.yml

```diff
@@ -55,6 +55,10 @@ pt-BR:
               uniqueness: 'O usuário já possui uma alocação em progresso'
             hourly_rate:
               invalid_currency: Moeda inválida
+        contribution:
+          attributes:
+            rejected_reason:
+              must_be_blank: "Não pode ser preenchido se o status for %{state}"
```

Comment:
Shouldn't we have these entries for `locales/en-US/models.yml` as well?

Reply:
You're right! Added those configs into `locales/en-US/models.yml `.